pkg/reconciler/reconciler_test.go

```diff
@@ -473,6 +474,19 @@ func TestReconcilerComplete(t *testing.T) {
 		t.Log("Expected "+strconv.Itoa(Edges)+" edges, but found ", len(com.Edges))
 		t.Fatalf("Error: Reconciler Complete() not working as expected.")
 	} else {
-		t.Log("Reconciler Complete() working as expected")
+		t.Log("Correct number of edges and nodes")
+	}
+
+	// Verify some properties are set during BuildEdges on ConfigurationPolicies
+	configPolNode := ns.ByKindNamespaceName["ConfigurationPolicy"]["local-cluster"]["policy-namespace"]
+
+	missing := configPolNode.Properties["_missingResources"]
+	if missing != `[{"v":"v1","k":"Namespace","n":"nonexistent"}]` {
+		t.Fatal("Incorrect _missingResources; got", missing)
+	}
```

Comment:
This works, or you could also use the assert library.
```suggestion
	assert.Equal(t, configPolNode.Properties["_missingResources"], `[{"v":"v1","k":"Namespace","n":"nonexistent"}]`)
```